### ISSUE TITLE
Button: focus indicator does not meet color contrast on light background

### ISSUE THREAD

**Issue by danethurber** [pluralsight/design-system#1587]
![Screen Shot 2021-01-29 at 9 52 06 AM](https://user-images.githubusercontent.com/226356/106303802-cb544700-6217-11eb-8829-344ce7197cca.png)

**Comment by jaketrent** [comment]
Sweeting updating design in true-ups.

**Comment by sweeting** [comment]
Closed by #1593
